templates/views/views-view--localgov-sitewide-search--page.html.twig

```diff
@@ -0,0 +1,95 @@
+{#
```

Comment:
This looks to me to be identical to a standard `views-view.html.twig` file. If that's the case, we probably don't need to add it here at all.

Reply:
The difference between this template and the standard Drupal one is the view filters come before the header. This is so the search block appears at the top and the page and not under the 'Search results' heading.

Doing this in the template seemed the best way to achieve this. Are you aware of any other ways to swap page elements without CSS hacks?

Reply:
I'm happy with that approach. I have a slightly modified views template in my default setup, where I use a variable to set things like that, then the custom views template just calls the default template and sets the variable to `true`. You can see a short write-up of it here:
https://mark.ie/blog/printing-regions-in-views-in-different-places-using-the-same-template